web/theme/default/libraries/jquery-ui-touch-punch/README.md

```diff
@@ -0,0 +1,41 @@
+# jQuery UI Touch Punch
```

Comment:
We don't need to readme or the unminified js - just the `.min.js` will be enough.